resources/lang/de/mainLang.php

```diff
@@ -49,10 +49,11 @@
     'marketingFlyersPlacard'=> 'Marketing / Flyer / Plakate', //used in legend.blade.php
     'preSale'               => 'Vorverkauf',
     'others'                => 'Sonstiges',
-    'faceDone'              => 'Facebook Event erledigt',
-    'eventUrl'              => 'Event Seite',
-    'priceTickets'          => 'Preis Vorverkauf Student / Extern',
-    'price'                 => 'Preis Student / Extern',
+    'faceDone'              => 'Facebook Event erstellt',
+    'eventUrl'              => 'Facebook-URL',
```

Comment:
1. Input said that this field isn't bound to facebook explicitly, thats why i changed it to this name
2. avoid the '-' character direct in the string, this will cause layout problems on small devices because the browser doesn't know hot to split this -> Facebook - URL